dashboard/app/views/home/_homepage.html.haml

```diff
@@ -7,6 +7,7 @@
 - homepage_data[:stageExtrasScriptIds] = Script.stage_extras_script_ids
 - homepage_data[:topCourse] = @top_course
 - homepage_data[:isEnglish] = @is_english
+- homepage_data[:locale] = Script.locale_english_name_map[request.locale]
```

Comment:
another gripe that I'm not sure we should do anything about: I really want to move this to `LocaleHelper` module, but the method caches internally via class variables and I'm not sure you can set class variables inside a module.

Reply:
yeah, i had the same feeling. unfortunately, we can't use a module, but we could create a singleton class that holds and caches this data. that's very "un-rails," so i'm not sure what the best way to go about that would be